### ISSUE TITLE
our_curators tiles should pull the curators' organisations from their user profiles

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#1955]
Hi all,

One curator is changing role and organisation, and Susanna has asked as a matter of priority to show any curators' relationships to any EOSC clusters. To prevent multiple tickets and duplication of organisation data, the best way stay up to date is to have this information drawn directly from the database. As this is just info in a json file right now, I realise this will need some work. 

This is part of the work that we need as soon as possible. If that means modifying the json file this time for a straightforward fix/update, that's fine. But then this ticket is here for the longer-term fix.

If you let me know which json file has the info for the curators, I can add the appropriate organisations for our short-term fix today.

Thanks!

**Comment by ramGranell** [comment]
Ok, the .json file with the curator information is here, https://github.com/FAIRsharing/fairsharing.github.io/blob/master/src/data/communityCurationCohorts.json but this change that you want to now perform is not easy as all the system was created to obtain the data from the .json file instead of the database. We can discuss about it tomorrow.

**Comment by allysonlister** [comment]
Yes, I've added it to our agenda for tomorrow. For the short term, I will just make the changes directly to the json file.
